x-pack/plugins/security_solution/public/management/pages/host_isolation_exceptions/store/reducer.test.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { UserChangedUrl } from '../../../../common/store/routing/action';
+import { HostIsolationExceptionsPageState } from '../types';
+import { initialHostIsolationExceptionsPageState } from './builders';
+import { HOST_ISOLATION_EXCEPTIONS_PATH } from '../../../../../common/constants';
+import { hostIsolationExceptionsPageReducer } from './reducer';
+import { getCurrentLocation } from './selector';
+
+describe('Host Isolation Exceptions Reducer', () => {
```

Comment:
A test for `hostIsolationExceptionsPageDataChanged` is missing

Reply:
I skipped it. The reducer is just returning the data of the payload, that'd be testing redux